src/Elliptic/Executables/Xcts/SolveXcts.hpp

```diff
@@ -0,0 +1,272 @@
+#include "Elliptic/DiscontinuousGalerkin/DgElementArray.hpp"
+#include "Elliptic/Systems/Xcts/FirstOrderSystem.hpp"
+#include "Elliptic/Tags.hpp"
+#include "Elliptic/Triggers/EveryNIterations.hpp"
+#include "IO/Observer/Actions/RegisterEvents.hpp"
+#include "IO/Observer/Helpers.hpp"
+#include "IO/Observer/ObserverComponent.hpp"
+#include "NumericalAlgorithms/Convergence/Tags.hpp"
+#include "Options/Options.hpp"
+#include "Parallel/Actions/SetupDataBox.hpp"
+#include "Parallel/Actions/TerminatePhase.hpp"
+#include "Parallel/GlobalCache.hpp"
+#include "Parallel/InitializationFunctions.hpp"
+#include "Parallel/PhaseDependentActionList.hpp"
+#include "Parallel/Reduction.hpp"
+#include "Parallel/RegisterDerivedClassesWithCharm.hpp"
+#include "ParallelAlgorithms/Events/ObserveErrorNorms.hpp"
+#include "ParallelAlgorithms/Events/ObserveFields.hpp"
+#include "ParallelAlgorithms/EventsAndTriggers/Actions/RunEventsAndTriggers.hpp"
+#include "ParallelAlgorithms/Initialization/Actions/AddComputeTags.hpp"
+#include "ParallelAlgorithms/Initialization/Actions/RemoveOptionsAndTerminatePhase.hpp"
+#include "ParallelAlgorithms/LinearSolver/Gmres/Gmres.hpp"
+#include "ParallelAlgorithms/LinearSolver/Tags.hpp"
+#include "ParallelAlgorithms/NonlinearSolver/NewtonRaphson/NewtonRaphson.hpp"
+#include "PointwiseFunctions/AnalyticData/AnalyticData.hpp"
+#include "PointwiseFunctions/AnalyticData/Xcts/AnalyticData.hpp"
+#include "PointwiseFunctions/AnalyticData/Xcts/Binary.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/Tags.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/Xcts/AnalyticSolution.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/Xcts/Flatness.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/Xcts/Kerr.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/Xcts/Schwarzschild.hpp"
+#include "Utilities/Blas.hpp"
+#include "Utilities/ErrorHandling/FloatingPointExceptions.hpp"
+#include "Utilities/Functional.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace SolveXcts::OptionTags {
+struct NonlinearSolverGroup {
+  static std::string name() noexcept { return "NonlinearSolver"; }
+  static constexpr Options::String help = "The iterative nonlinear solver";
+};
+struct NewtonRaphsonGroup {
+  static std::string name() noexcept { return "NewtonRaphson"; }
+  static constexpr Options::String help =
+      "Options for the Newton-Raphson nonlinear solver";
+  using group = NonlinearSolverGroup;
+};
+struct LinearSolverGroup {
+  static std::string name() noexcept { return "LinearSolver"; }
+  static constexpr Options::String help =
+      "The iterative Krylov-subspace linear solver";
+};
+struct GmresGroup {
+  static std::string name() noexcept { return "Gmres"; }
+  static constexpr Options::String help = "Options for the GMRES linear solver";
+  using group = LinearSolverGroup;
+};
+}  // namespace SolveXcts::OptionTags
+
+/// \cond
+struct Metavariables {
+  static constexpr size_t volume_dim = 3;
+  static constexpr int conformal_matter_scale = 0;
```

Comment:
I could not find the documentation for `Xcts::FirstOrderSystem` in order to look up what this int is...

Reply:
It's here https://spectre-code.org/structXcts_1_1FirstOrderSystem.html

Reply:
strange; doesn't show up on your branch when I build it...